engine/access/state_stream/backend/handler_test.go

```diff
@@ -0,0 +1,424 @@
+package backend
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	pb "google.golang.org/genproto/googleapis/bytestream"
+	"google.golang.org/grpc"
+
+	jsoncdc "github.com/onflow/cadence/encoding/json"
+	access "github.com/onflow/flow/protobuf/go/flow/executiondata"
```

Comment:
There was a change recently that fixed this namespace
```suggestion
	"github.com/onflow/flow/protobuf/go/flow/executiondata"
```